index.html

```diff
@@ -150,6 +150,33 @@
     </div>
 </section>
 
+<section>
+    <div class="pink-stripe"></div>
+    <div class="header-text">What we use</div>
+    <div class="blue-stripe no-margin-bottom"></div>
+    <div class="section-body  no-margin">
+        <div class="tech-body">
+            <div class="tech-container">
+                <img class="tech" src="./assets/images/logos/python-logo-full.png" alt="python-logo.png">
+            </div>
+            <div class="tech-container">
+                <div class="padding-container">
+                    <img class="tech" src="./assets/images/logos/django-logo-full.png" alt="django-logo.png">
+                </div>
+            </div>
+            <div class="tech-container">
+                <img class="tech" src="./assets/images/logos/vue-logo-full.png" alt="vue-logo.png">
+            </div>
+            <div class="tech-container">
+                <img class="tech" src="./assets/images/logos/heroku-logo-full.png" alt="heroku-logo.png">
+            </div>
```

Comment:
Should swap these tags. The composition of the tags is weighted to the right.
![image](https://user-images.githubusercontent.com/16579931/96513457-ff7e9b80-1216-11eb-83b1-11a527dd464f.png)
When in this order it looks a little more balanced.
![image](https://user-images.githubusercontent.com/16579931/96513329-d2ca8400-1216-11eb-8e7f-11f4c0a9dfda.png)

Reply:
Great input